### PR TITLE
libfilezilla: bump version

### PR DESCRIPTION
**Pull request by miqlas**

### CODE REVIEW

**Comment by miqlas** [comment]
Broken, Do not merge yet.

File: dev-libs/libfilezilla/libfilezilla-0.9.1.recipe [hunk 1]
```diff
@@ -7,13 +7,13 @@ extremely efficient
...
-COPYRIGHT="2015-2016 Tim Kosse"
+HOMEPAGE="https://lib.filezilla-project.org/"
+COPYRIGHT="2015-2017 Tim Kosse"
 LICENSE="GNU GPL v2"
 REVISION="1"
 SOURCE_URI="http://download.filezilla-project.org/libfilezilla/libfilezilla-$portVersion.tar.bz2"
-CHECKSUM_SHA256="41d02b3eb54be1b1fdab89104a28a47fb654465a8d9ad00446c221a27bd9800c"
-PATCHES="libfilezilla-$portVersion.patchset"
+CHECKSUM_SHA256="18b2391771f330cccab2c55a66197b9098f236e616f26f86326795b900913b1a"
+PATCHES="libfilezilla-0.9.0.patchset"
```
Comment (korli) [inline dev-libs/libfilezilla/libfilezilla-0.9.1.recipe]: Please rename the patchset instead.

**Comment by korli** [comment]
With your last change you can probably drop the patch set.

**Comment by miqlas** [comment]
Nope, it is a New patchset

**Comment by korli** [comment]
I see but what it does is definitely already done by the CFLAGS in the recipe.

**Comment by miqlas** [comment]
Tested, doesn't. Idk why.

-- Z.

2017.03.21. dátummal, 7:13 időpontban Jérôme Duval <notifications@github.com> írta:

**Comment by korli** [comment]
Hmm it's a cpp file, please use CPPFLAGS (for C and CPP files).

**Pull request merged**